share/picongpu/examples/AtomicPhysics/include/picongpu/param/speciesDefinition.param

```diff
@@ -32,6 +32,7 @@
 
 #pragma once
 
+#include "picongpu/simulation_defines.hpp"
```

Comment:
this wile dows not exits anymore and it would be also wrong to include this within a param file